### ISSUE TITLE
Validation not Working

### ISSUE THREAD

**Issue by ameer1234567890** [GoogleChromeLabs/bubblewrap#627]
When running `bubblewrap validate url=https://www.google.com`, following error occurs:

```
cli ERROR Error calling the PageSpeed Insights API: Error: Failed to run the PageSpeed Insight report for https://www.googleapis.com/pagespeedonline/v5/runPagespeed?url=https%3A%2F%2Fhome.ameer.io%2F&category=performance&category=pwa&category=accessibility&strategy=mobile. Server responded with status 429
```

**Comment by Geri1590** [comment]
Same happens when trying to access a server protected by a VPN.

**Comment by andreban** [comment]
Validation uses a PageSpeed Insights endpoint. If the endpoint is unreacheable, the call will fail. It doesn't stop the project from building though.

@nohe427 I' wondering about just removing validation from the build step. Thoughts?

**Comment by andreban** [comment]
#675 removes the validation from the build step. We should also consider deprecating the validate command and package.

**Comment by andreban** [comment]
Closed via #675
